native/dotnet-sdk-8.0/digests

```diff
@@ -0,0 +1,3 @@
+dotnet-sdk-8.0.204-linux-x64.tar.gz SHA1 7f31552610c9139db5be0ace78d3378242128720
```

Comment:
Here are the digests for .301:
dotnet-sdk-8.0.301-linux-x64.tar.gz SHA1 C6A978B16E2462CE91446D987CCA349C875DB5D3
dotnet-sdk-8.0.301-linux-x64.tar.gz SHA256 D237CF0E8865F51EB9634DF32A8BCFB1DD95DAC95C0D16B5D599DF867AB16C27
dotnet-sdk-8.0.301-linux-x64.tar.gz MD5 D91E94323D8DB5551C9A5179162F2532